src/dmtcp_launch.cpp

```diff
@@ -146,6 +147,8 @@ static const char *theUsage =
   "              Skip NOTE messages; if given twice, also skip WARNINGs\n"
   "  --coord-logfile PATH (environment variable DMTCP_COORD_LOG_FILENAME\n"
   "              Coordinator will dump its logs to the given file\n"
+  "  --mpi\n"
+  "              Set it to enable MPI-specific flags.\n"
```

Comment:
It his is `to enable MPI-specific flags`, then what are the other MPI-specific flags?  Or did you want to say `to enable MPI/MANA` ?

Or is this intended as:  `Set it to enable '--mpi' flag for MPI/MANA` ?

Or is it:  `Set it to enable 'dmtcp_launch --mpi'` ?

Reply:
Fixed.